### DISCUSSION TITLE
怎么提升表格识别的准确度

### DISCUSSION THREAD

**Discussion by Andriy7** [PaddlePaddle/PaddleOCR#14335] [Q&A]
使用ppstructure不能获得很好的表格结果,求助怎么进行优化?
![Uploading table.jpg…]()

**Comment by GreatV** [comment]
根据您的问题和提供的文档,我总结以下几点可以帮助提升表格识别的准确度:

1. 优化模型选择:
- 使用PP-StructureV2中的SLANet模型,它较TableRec-RARE在精度上提升了4.7%,TEDS提升2%
- 搭配最新的PP-OCRv3文字检测和识别模型

2. 调整模型参数:
- table_max_len: 根据实际表格大小设置合适的长度,不要盲目增大
- batch_size: 可以尝试调整batch_size大小
- 学习率: 可以适当调小学习率,如设为原来的0.05倍

3. 数据预处理:
- 对倾斜的表格图片进行矫正
- 提升图像分辨率和清晰度
- 去除图像噪声和干扰

4. 针对具体场景:
- 如果是复杂表格,建议重新训练模型
- 可以结合表格属性识别来判断表格难度,对难识别的表格重点处理
- 对于不换行标题等特殊情况,需要专门优化处理

5. 模型微调:
- 可以在预训练模型基础上,用自己的数据集进行fine-tuning
- 收集更多真实场景的数据进行训练

6. 后处理优化:
- 对识别结果进行校验和纠错
- 设置合理的文本过滤规则
- 针对特定业务场景制定后处理规则

建议您先从以上几个方面入手尝试优化。如果还不能解决问题,可以提供具体的错误示例,这样可以给出更有针对性的建议。

---
*Response generated by 🤖 [feifei-bot](https://github.com/GreatV/feifei) | claude-3-5-sonnet-latest*

**Comment by SWHL** [comment]
一是尝试其他开源表格识别模型，例如→ [TableStructureRec](https://github.com/RapidAI/TableStructureRec)
二是基于自己数据微调
三是尝试LLM系列的，例如[Qwen2-VL](https://github.com/QwenLM/Qwen2-VL)系列，或者[mPLUG-DocOwl](https://github.com/X-PLUG/mPLUG-DocOwl)
